helmfile/charts/vulnerability-exporter/templates/configmap.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vulnerability-exporter-cronjob
+data:
+  cronjob: |
+    ### cron file for running the export script
+    {{ .Values.vulnerabilityExporter.schedule }} root sh /scripts/convert-vulnerabilityreports.sh
+    # An empty line is required at the end of this file for a valid cron file.
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vulnerability-exporter-script
+data:
+  convert-vulnerabilityreports.sh: |
+    #!/bin/sh
+
+    APISERVER=https://kubernetes.default.svc
+    SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
+    NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
+    TOKEN=$(cat ${SERVICEACCOUNT}/token)
+    CACERT=${SERVICEACCOUNT}/ca.crt
+    curl --cacert ${CACERT} --header "Authorization: Bearer ${TOKEN}" -X GET ${APISERVER}/apis/aquasecurity.github.io/v1alpha1/vulnerabilityreports > /textfile-collector/tmp-reports.json
```

Comment:
Don't you get the 

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  ```
  
  if you don't have the `--silent` flag?
  Maybe it doesn't matter but it would be an easy addition

Reply:
I don't think it matters, but I added it anyway (I suppose it's fewer lines it has to parse in the text file it creates, at least).